Standard.Image is broken on macos/aarch64

https://github.com/enso-org/enso/actions/runs/13002835632/job/36264581465#step:8:1287

Broken since https://github.com/enso-org/enso/pull/12021.

[Related](https://github.com/enso-org/enso/actions/runs/13044872444/job/36393690033#step:8:3234):
```
 INFO ide_ci::program::command: enso ⚠️ Execution finished with an error: No opencv_java470 in java.library.path
 INFO ide_ci::program::command: enso ⚠️         at <java> org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.NativeLibrarySupport.loadLibraryRelative(NativeLibrarySupport.java:136)
 INFO ide_ci::program::command: enso ⚠️         at <java> java.base@21.0.2/java.lang.ClassLoader.loadLibrary(ClassLoader.java:106)
 INFO ide_ci::program::command: enso ⚠️         at <java> java.base@21.0.2/java.lang.Runtime.loadLibrary0(Runtime.java:916)
 INFO ide_ci::program::command: enso ⚠️         at <java> java.base@21.0.2/java.lang.System.loadLibrary(System.java:2063)
 INFO ide_ci::program::command: enso ⚠️         at <java> nu.pattern.OpenCV$SharedLoader.<init>(OpenCV.java:200)
 INFO ide_ci::program::command: enso ⚠️         at <java> nu.pattern.OpenCV$SharedLoader.<init>(OpenCV.java:189)
 INFO ide_ci::program::command: enso ⚠️         at <java> nu.pattern.OpenCV$SharedLoader$Holder.<clinit>(OpenCV.java:261)
 INFO ide_ci::program::command: enso ⚠️         at <java> nu.pattern.OpenCV$SharedLoader.getInstance(OpenCV.java:265)
 INFO ide_ci::program::command: enso ⚠️         at <java> nu.pattern.OpenCV.loadShared(OpenCV.java:183)
 INFO ide_ci::program::command: enso ⚠️         at <java> org.enso.image.data.Matrix.<clinit>(Matrix.java:10)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Matrix.type.zeros<arg-1>(Matrix.enso:42:23-61)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Matrix.type.zeros(Matrix.enso:42:9-62)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Matrix_Spec.add_specs.Matrix_Spec.add_specs(src/Data/Matrix_Spec.enso:17:17-32)
 INFO ide_ci::program::command: enso ⚠️         at <enso> case_branch(Suite.enso:37:17-32)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Suite_Builder.group(Suite.enso:35-42)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Matrix_Spec.add_specs(src/Data/Matrix_Spec.enso:16-115)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Main.main.suite(src/Main.enso:12:9-43)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Test.build(Test.enso:24:9-33)
 INFO ide_ci::program::command: enso ⚠️         at <enso> Main.main(src/Main.enso:10-13)
 INFO ide_ci::program::command: enso ⚠️ No opencv_java470 in java.library.path
```

[and](https://github.com/enso-org/enso/actions/runs/13044872444/job/36393690512#step:8:5141)
```
 INFO ide_ci::program::command: sbt ℹ️ [error] Test org.enso.interpreter.test.NativeLibraryFinderTest.standardImageShouldHaveNativeLib failed: java.lang.AssertionError: 
 INFO ide_ci::program::command: sbt ℹ️ [error]   os.name: Mac OS X
 INFO ide_ci::program::command: sbt ℹ️ [error]   os.arch: aarch64
 INFO ide_ci::program::command: sbt ℹ️ [error]   Mapped library name: libopencv_java470.dylib
 INFO ide_ci::program::command: sbt ℹ️ [error]   Contents of Standard.Image native library dir:
 INFO ide_ci::program::command: sbt ℹ️ [error] , took 2.064 sec
 INFO ide_ci::program::command: sbt ℹ️ [error]     at org.enso.interpreter.test.NativeLibraryFinderTest$PrintSystemInfoRule$1.evaluate(NativeLibraryFinderTest.java:84)
 INFO ide_ci::program::command: sbt ℹ️ [error]     ...
 INFO ide_ci::program::command: sbt ℹ️ [error] Caused by: java.lang.AssertionError: There should be just single native lib in Standard.Image
 INFO ide_ci::program::command: sbt ℹ️ [error] Expected: is <1>
 INFO ide_ci::program::command: sbt ℹ️ [error]      but: was <0>
 INFO ide_ci::program::command: sbt ℹ️ [error]     at org.enso.interpreter.test.NativeLibraryFinderTest.standardImageShouldHaveNativeLib(NativeLibraryFinderTest.java:49)
 INFO ide_ci::program::command: sbt ℹ️ [error]     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
 INFO ide_ci::program::command: sbt ℹ️ [error]     at java.lang.reflect.Method.invoke(Method.java:580)
 INFO ide_ci::program::command: sbt ℹ️ [error]     ...
 INFO ide_ci::program::command: sbt ℹ️ [info] Test run org.
```

--- a/project/StdBits.scala
+++ b/project/StdBits.scala
@@ -198,7 +198,9 @@ object StdBits {
     */
   private def arch(): String = {
     val arch = System.getProperty("os.arch").toLowerCase(Locale.ENGLISH)
-    arch.replace("amd64", "x86_64")
+    arch
+      .replace("amd64", "x86_64")
+      .replace("aarch64", "ARMv8")
   }
 
   private def updateDependency(

